docs/readmes/basics/prerequisites.md

```diff
@@ -60,7 +60,7 @@ Development can occur from multiple OS's, where **macOS** and **Ubuntu** are **e
 1. Install the following tools
    1. [Docker](https://docs.docker.com/engine/install/ubuntu/) and [Docker Compose](https://docs.docker.com/compose/install/)
    2. [VirtualBox](https://www.virtualbox.org/wiki/Linux_Downloads)
-   3. [Vagrant](https://www.vagrantup.com/downloads)
+   3. [Vagrant](https://www.vagrantup.com/downloads) (Install by downloading the `.deb` file. Installing via the command line using `apt-get` can currently cause an issue with OpenSSL.)
```

Comment:
What's the issue that gets caused with OpenSSL? Could we potentially link to an external page somewhere that explains the issue so the problem can be tracked?

Reply:
https://github.com/hashicorp/vagrant/issues/12751

I added this link. They also don't specify what the issue is exactly but they found the same fix.